src/effects/builtin/semiparametriceq4knobeffect.cpp

```diff
@@ -0,0 +1,157 @@
+#include "effects/builtin/semiparametriceq4knobeffect.h"
+
+namespace {
+static const double kMinCorner = 13;    // Hz
```

Comment:
Same here. Maybe add a shared base class to reduce duplication?

Reply:
good idea, done: 33f500de0e14dfffcfc8ac700d1f71c395f08439